src/main/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/JaggaerBuilder.java

```diff
@@ -208,13 +221,24 @@ public static JaggaerBasicValue buildJaggaerBasicValue(String value) {
         return buildJaggaerBasicValue(value, null);
     }
     
+    public static JaggaerBasicValue buildJaggaerBasicValue(boolean value, JaggaerBooleanToStringType stringType) {
+        return buildJaggaerBasicValue(value, null, stringType);
```

Comment:
In my final review of the PR, I saw this method call. I am having some difficulty with using a null parameter value for an item that is defined as a String BUT screams Java Boolean based on its naming convention. 

@jhulslander and I spoke about this issue. It is a Jaggaer"ism" that has to deal with setting up the DTOs for the data transfer. Since two other developer's reviewed this code prior to me without raising an objection and the fact that it is a fundamental part of the Jaggaer processing, I will be approving this PR to move forward. We will discuss as a team whether we need to address anything with regard to this method or the class that uses it in a future work ticket. 

At a minimum, I would like to see a comment in class edu.cornell.kfs.module.purap.jaggaer.supplier.xml.Active listing all acceptable values as well as a statement that the attribute's value can be null which prevents the data from being placed in the DTO. My concern is that the only place this specific detailed information is located is in the Jaggaer documentation the lead project developer is using to implement this functionality and we are going to lose the WHY we implemented this the way we did.

Reply:
Just a thought: Would an `@XmlJavaTypeAdapter` be helpful here? That way, you could have the DTO field be defined as a boolean (or some object wrapping a boolean), and then the conversion between the boolean value and the yes/no/true/false strings would be handled by the adapter class.

Reply:
@cah292 A type adapter might help with active tag value, but would take some rework to get it correct.  There is one Active POJO but the valid values depend on different scope, inside supplier tag versus inside location, versus inside address.  Nancy's concern was about the isChanged String, not the boolean conversion to String logic.  The isChanged attributed is included in many Jaggaer POJOs, but aren't used in the supplier upload XML.

Reply:
If we should switch to type adapters, I think that should be done on a focused user story, as it will get pretty complex

Reply:
over night I thought of some ways to clear up some confusion with this. let me work on that